interview-questions.md

```diff
@@ -8,48 +8,60 @@ Then, research the question to expand on your answer. Even if you feel you have
 
```

Comment:
Consider what you want to incorporate into your interviews. Create a flow for yourself and share it on next week assessment. Things I am looking for more content to include an example of how you used the topic. Think making a conversation with the interviewer.